### DISCUSSION TITLE
When clicking on a title to jump to its corresponding content, the page does not display the title itself.

### DISCUSSION THREAD

**Discussion by Dion4cen** [squidfunk/mkdocs-material#5509] [Q&A] [answered]
Description：
"Whenever I click on a title in the sidebar on the right, the page jumps to the corresponding section, but it goes directly to the content below the title without displaying the title itself. You can refer to [this page](https://docs.arducam.com/Raspberry-Pi-Camera/Native-camera/Troubleshooting/#troubleshooting_1) for example.
I want the page to jump to the corresponding section when I click on a title in the sidebar on the right, and have the title displayed at the top of the page.

Troubleshooting I tried:
Modify the yam file, add css file in the project etc, all of them do not work.

Is there anyone else meeting this phenomenom before? Or can someone here give me some advice on how to solve this problem.
Any help would be appreciated :-)

**Comment by kamilkrzyskow** [comment, marked as answer]
I don't know if you can influence the end position after clicking on the table of contents 🤔 
You've added an additional header to the page and moved the default logo a bit down, and it hides the name of the headings.
By removing the additional logo and turning off the extra css, the titles are always visible.

https://github.com/squidfunk/mkdocs-material/assets/34622465/bb2dfe1f-45c2-4f67-87d5-af4f40e6d729

**Comment by Dion4cen** [comment]
Oh, thanks for your prompt reply, you help me a lot!!!!! 
I'm a rookie of mkdocs for material, but I'm so intersted in it and I will keep learning it :-)))

**Reply by kamilkrzyskow** [reply]
I haven't tried it myself, but it seems there are different ways how to deal with fixed headers. Give them a try:
https://stackoverflow.com/questions/4086107/fixed-page-header-overlaps-in-page-anchors
https://stackoverflow.com/questions/10732690/offsetting-an-html-anchor-to-adjust-for-fixed-header
